diff --git a/node_modules/@react-native-community/image-editor/ios/RNCImageEditor.m b/node_modules/@react-native-community/image-editor/ios/RNCImageEditor.m
index 7c2bede..7b321f9 100644
--- a/node_modules/@react-native-community/image-editor/ios/RNCImageEditor.m
+++ b/node_modules/@react-native-community/image-editor/ios/RNCImageEditor.m
@@ -17,10 +17,10 @@
 #import <React/RCTImageStoreManager.h>
 #import "RNCFileSystem.h"
 #import "RNCImageUtils.h"
-#if __has_include(<RCTImage/RCTImageUtils.h>)
-#import <RCTImage/RCTImageUtils.h>
+#if __has_include(<React/RCTImageUtils.h>)
+#import <React/RCTImageUtils.h>
 #else
-#import "RCTImageUtils.h"
+#import <React/RCTImageUtils.h>
 #endif
 
 @implementation RNCImageEditor
